bin/brew

```diff
@@ -13,6 +13,8 @@ fi
 BREW_FILE_DIRECTORY="$(chdir "${0%/*}" && pwd -P)"
 HOMEBREW_BREW_FILE="$BREW_FILE_DIRECTORY/${0##*/}"
 
+export HOMEBREW_PREFIX=$(cd $(dirname $(dirname $HOMEBREW_BREW_FILE)) && pwd -P)
```

Comment:
`chdir` ensures that the output is sent to `/dev/null` (in case of weird shell aliases/settings)